applications/DEMApplication/tests/test_post_process.py

```diff
@@ -46,9 +46,12 @@ def test_gid_printing_many_results(self):
         model = KratosMultiphysics.Model()
         auxiliary_functions_for_tests.CreateAndRunStageInSelectedNumberOfOpenMPThreads(TestPostProcessClass1, model, parameters_file_name, 1)
 
+    @classmethod
     def tearDown(self):
```

Comment:
I think this does not work properly
There is a special fct for this purpose: https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDownClass

Reply:
Thanks for the info, I change it.